### DISCUSSION TITLE
Photography Help Wanted

### DISCUSSION THREAD

**Discussion by Jyers** [Jyers/Marlin#536] [General Discussion]
Hey everyone! As always thanks for all the activity and involvement in this community, I still can't believe how much this firmware has grown over the last couple months. 

I am posting to ask for someone's assistance. As is probably painfully aware, I can not manage to get good pictures of the V2 LCD. No matter what I do it's either exposed wrong, or the viewing angle is weird or its distorted. No matter what it just isn't good. 

I'm planning on making a few posts around the various social groups where this firmware has become known and I would love to have some good images of the screen and of the different menu's so that people can understand what they're missing. Here are the main pictures I think would be the best to showcase the firmware:
* Main Menu
* Print Screen
* Advanced/Visual Menus
* Z Offset Menu
* Manual Leveling (tramming)
* Manual Mesh Leveling
* UBL Mesh Viewer
* Color Settings Menu
* The Main Menu using a few different color themes

Of course if you think of any other menus or pictures that show off what makes this firmware what it is, of course all are welcome. 

So if you happen to have a decent camera and are able to get good pictures of the screen please let me know, and hopefully I can get some better pictures not only for the posts but for the wiki pages too!

**Comment by Cal-LF** [comment]
Where can the photos be posted? I can send a zipped set. I don't have BL Touch or use the UBL Mesh. I can help but with these limitations.

**Reply by Jyers** [reply]
You can directly attach the zip to a post on this discussion if you want, or if you'd rather you can send them to me at Jyers.Marlin@gmail.com

**Comment by Cal-LF** [comment]
Check these over. I will not be offended if you want to use a set from other contributors.
[Menus.zip](https://github.com/Jyers/Marlin/files/6339037/Menus.zip)

**Reply by Jyers** [reply]
These look great! I'll definitely use some, I might just need to get some from others with the BL Touch menus and such

**Comment by rorywquin** [comment]
I'm busy this week. I'll follow this and if you don't get what you need, I'll do some next week. I assume you just want to see the screen clearly. I'm surprised at the poor response to your request as there must be 100s of people benefitting from your work.

**Reply by Jyers** [reply]
That would be great! Thank you

**Reply by frozentimeimages** [reply]
I think the reason for the poor response is that most of us 3D printers aren't members of GitHub, so didn't know about the request. I only just found it by chance, too late apparently. But now I joined just to be able to help in future.
Great work you do Jake !

**Reply by t3chguy** [reply]
Guess he should have put a "HELP WANTED" ad on the settings page in the firmware :P

**Comment by gothcha** [comment]
Hi Jacob,

I hope these will be useful, if there are any I have missed just let me know. V1.3.1 is working a treat thank you, I just love it.

Sent in a few sections ...
[Part 1.zip](https://github.com/Jyers/Marlin/files/6351929/Part.1.zip)
[Part 2.zip](https://github.com/Jyers/Marlin/files/6351931/Part.2.zip)
[Part 3.zip](https://github.com/Jyers/Marlin/files/6351934/Part.3.zip)
[Part 4.zip](https://github.com/Jyers/Marlin/files/6351936/Part.4.zip)

**Reply by Jyers** [reply]
These look great! Thank you so much, glad your enjoying the firmware!

**Comment by gothcha** [comment]
No problem - a few more for you
[Part 5.zip](https://github.com/Jyers/Marlin/files/6352186/Part.5.zip)
[Part 6.zip](https://github.com/Jyers/Marlin/files/6352188/Part.6.zip)

A few more ..

**Reply by gothcha** [reply]
Some for the mesh tuning steps
[Part 7.zip](https://github.com/Jyers/Marlin/files/6352282/Part.7.zip)

**Reply by gothcha** [reply]
Just a thought - The photos I sent you are using the  @AngleV top level icons, but I notice your repo DWIN_SET is still using the stock versions. Are you planning to use these new icons?
